cmd/flag/open_stream_flags.go

```diff
@@ -17,15 +17,32 @@ package flag
 
 import (
 	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	defaultEnableAutoClose = false
+	defaultAutoCloseDelay  = 5 * time.Second
+	defaultAutoCloseTime   = "2006-01-02'T'15:04:05Z"
```

Comment:
I don't think there should be a default value for this, and that there should be an error thrown if auto close is enabled and no timestamp is provided.

Reply:
I'll change this.

Reply:
If danada's comment relates to the question you asked me yesterday, then I would also suggest that the timestamp does not have a default. If you forget to specify it, esp. if using default delay too then (assuming that you, as I will, start up some time before the pass begins) then that could well mean you've made a mistake/oversight. 
I'd say 'ok' to the 5s default, but I would say it would be safer to _not_ allow a default timestamp. Only my opinion :)